### PR TITLE
Add code-variable extension

### PR DESCRIPTION
**Pull request by Hongbusi**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Hongbusi** [comment]
CodeVariable returns a better variable from Chinese to English. Support small camel case, large camel case, constant, underscore, dash.

File: extensions/code-variable/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+![常量命名法](./metadata/code-variable-5.png)
+
+## Special Thanks
+
+This project is based on and inspired by Alfred extension [CodeVar](https://github.com/xudaolong/CodeVar).
+
+## License
+
+[MIT](./LICENSE) License © 2023 [Hongbusi](https://github.com/Hongbusi)
```
Comment (pernielsentikaer) [inline extensions/code-variable/README.md:35]: At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Do the users of this extension know hoe to use it? I have no clue at all so we might need to describe it better 😊

**Comment by Hongbusi** [comment]
Sincerely recommend changing the extension system, even if the user experience makes me feel great, releasing an extension will make me lose interest in continuing to use it.

**Pull request closed without merging**